### PR TITLE
Updated now that we supported TSA!

### PR DESCRIPTION
**Pull request by islubin**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tenant-apps-catalog-teams.md [hunk 1]
```diff
@@ -25,20 +25,20 @@ You can use the Microsoft Teams Tenant Apps Catalog to test and distribute line-
 
 The Teams Tenant Apps Catalog lets you distribute line-of-business applications that were built specifically for your organization and that you rely on to complete critical business functions.
 
-To publish apps for your organization, sign in to your Teams client using your global admin credentials and then follow the instructions below.
+To publish apps for your organization, sign in to your Teams client using an account with the global admin or teams service admin roles and then follow the instructions below.
```
Comment (JohanFreelancer9) [inline Teams/tenant-apps-catalog-teams.md]:
```suggestion
To publish apps for your organization, sign in to your Teams client using an account with the global admin or Teams service admin roles and then follow the instructions below.
```

File: Teams/tenant-apps-catalog-teams.md [hunk 2]
```diff
@@ -25,20 +25,20 @@ You can use the Microsoft Teams Tenant Apps Catalog to test and distribute line-
...
 
 ## Publish an app in the Tenant Apps Catalog from the Teams client
 
 > [!NOTE]
-> You need to be signed in to the Microsoft Teams client using your global admin credentials to publish apps for your organization.
+> You need to be signed in to the Microsoft Teams client using an account that has either the global admin or teams service admin role enabled to publish apps for your organization.  [Learn more about the admin roles for managing teams](https://docs.microsoft.com/en-us/MicrosoftTeams/using-admin-roles).
```
Comment (JohanFreelancer9) [inline Teams/tenant-apps-catalog-teams.md]:
```suggestion
> You need to be signed in to the Microsoft Teams client using an account that has either the global admin or Teams service admin role enabled to publish apps for your organization.  [Learn more about the admin roles for managing teams](https://docs.microsoft.com/en-us/MicrosoftTeams/using-admin-roles).
```

Comment (JohanFreelancer9) [inline Teams/tenant-apps-catalog-teams.md]:
```suggestion
> You need to be signed in to the Microsoft Teams client using an account that has either the global admin or teams service admin role enabled to publish apps for your organization. [Learn more about the admin roles for managing teams](https://docs.microsoft.com/en-us/MicrosoftTeams/using-admin-roles).
```

File: Teams/tenant-apps-catalog-teams.md [hunk 3]
```diff
@@ -25,20 +25,20 @@ You can use the Microsoft Teams Tenant Apps Catalog to test and distribute line-
...
+A Teams app package is created by using [Teams App Studio](https://docs.microsoft.com/microsoftteams/platform/get-started/get-started-app-studio). Once you have the app package, you can add it to the enterprise app catalog. While all users in the tenant can view the app catalog, only global admins and teams service admins have the ability to publish and manage it.
 
 ### Go to the Tenant Apps Catalog
 
-Start the Microsoft Teams client and sign-in using your global admin credentials. From the Microsoft Teams Store, select the new section named for your specific organization (in this example, Contoso). Users in your organization can view apps in the catalog and install them for teams of which they are a member.
+Start the Microsoft Teams client and sign-in using your global or teams service admin credentials. From the Microsoft Teams Store, select the new section named for your specific organization (in this example, Contoso). Users in your organization can view apps in the catalog and install them for teams of which they are a member.
```
Comment (JohanFreelancer9) [inline Teams/tenant-apps-catalog-teams.md]:
```suggestion
Start the Microsoft Teams client and sign in using your global or teams service admin credentials. From the Microsoft Teams Store, select the new section named for your specific organization (in this example, Contoso). Users in your organization can view apps in the catalog and install them for teams of which they are a member.
```

File: Teams/tenant-apps-catalog-teams.md (hunk 2 above)
Comment (nenonix) [inline Teams/tenant-apps-catalog-teams.md]:
```suggestion
>You need to be signed in to the Microsoft Teams client with an account that has either the global admin or teams service admin role enabled to publish apps for your organization. Learn more about [using administrator roles to manage Teams](https://docs.microsoft.com/MicrosoftTeams/using-admin-roles).
```

**Comment by LolaJacobsen** [comment]
@islubin - please review (and accept or reject) the interim commits. Then we can merge.

**Comment by islubin** [comment]
@LolaJacobsen these changes look good to me.

**Comment by kenwith** [comment]
@islubin - you have to be the one to click the Commit for each suggestion since it is committing to your branch in your fork you are the only one with permissions. :)

**Pull request merged**